src/core/server/graph/resolvers/NewCommentersConfiguration.ts

```diff
@@ -3,7 +3,7 @@ import * as settings from "coral-server/models/settings";
 import {
   GQLMODERATION_MODE,
   GQLNewCommentersConfigurationTypeResolver,
-} from "core/client/framework/schema/__generated__/types";
+} from "coral-server/graph/schema/__generated__/types";
```

Comment:
This is a gotcha that will get you bad as it is invisible until you do a production build and start it up.

Make sure we always use `coral-server/` types when we are in server code. There are types that are nearly (or sometimes exactly) identical for the client side types as well, and if you let VSCode "auto-import" this for you, it will pick the client side types first because `c` in `client` comes before `s` in `server` alphabetically.

You all probably already know to do this, but double check your types! VSCode autocomplete with getcha!

Reply:
I agree that we should look into seeing if the linter can catch this for us!